spk/stockfish/Makefile

```diff
@@ -0,0 +1,15 @@
+SPK_NAME = stockfish
+SPK_VERS = 11
+SPK_ICON= src/sf_icon.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = Guillaume
+DESCRIPTION = Stockfish is a strong open source chess engine 
+DISPLAY_NAME = Stockfish 
+STARTABLE=no
```

Comment:
I propose to start engine as a DSM service too.

Reply:
It is not startable. At least I don't know how to start as service. The binary starts an interactive console when executed without any parameter. Therefore "STARTABLE=no" is correct.